specification/applicationinsights/resource-manager/readme.md

```diff
@@ -216,6 +216,18 @@ These settings apply only when `--tag=package-preview-2021-03-only` is specified
 input-file:
   - Microsoft.Insights/preview/2021-03-03-preview/diagnosticServicesToken_API.json
 ```
+
+### Tag: package-2021-03-08
+
+These settings apply only when `--tag=package-2021-03-08` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-03-08'
+input-file:
+  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbooks_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
+```
+
```

Comment:
```suggestion
### Tag: package-2021-03-08-only

These settings apply only when `--tag=package-2021-03-08-only` is specified on the command line.

``` yaml $(tag) == 'package-2021-03-08-only'
input-file:
  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
  - Microsoft.Insights/stable/2021-03-08/workbooks_API.json
  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
```

```

Reply:
what does the -only suffix of the tag do? quick search in swagger wiki and github didn't show anything relevant.

Reply:
if a tag contains only part of API files, it should add `-only`.